src/Restivus/Extensions.cs

```diff
@@ -26,7 +30,11 @@ public static Uri FilterQueryParams(this Uri uri, params string[] keys)
             {
                 foreach (var key in keys)
                 {
+#if NETSTANDARD13
+                    if (queryParams.ContainsKey(key))
+#elif NET4_5
                     if (queryParams.AllKeys.Contains(key))
+#endif
```

Comment:
Also, what happens if neither `NETSTANDARD13` nor `NET4_5` are defined?
Does it just fail to compile?

That's most likely just fine, I'm just mostly curious.